int8 FlyStopped
int8 RobotStopped
